estate/models/estate.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from dateutil.relativedelta import *
+
+
+class Estate(models.Model):
+    _name = "estate_property"
```

Comment:
_name is defined with dots, not underscores + the class name should reflect the name
```suggestion
class EstateProperty(models.Model):
    _name = "estate.property"
```